### PR TITLE
[next] Fixes #1560 - Porting AboutFooter to NextJS

### LINKED ISSUE #1560: [next] Porting AboutFooter to NextJS
**What would you like to be added**:
The AboutFooter component ported over to NextJS using TS

**Why would you like this to be added**:
Part of the porting of the Telescope Front-end. Originally issued as part of #1454 but was split away since there were some components missing for the rendering of the AboutFooter component. Those were LogoIcon (now Logo), and GitHubContributorCard. Both are now merged on master so this component can now finally be handled

### PR DESCRIPTION
**Pull request by HyperTHD**
## Issue This PR Addresses
This PR fixes #1560 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

This PR is apart of #1316 which is the transitioning of the Telescope Front-end from Gatsby to NextJS. This PR addresses the porting of the AboutFooter component. 

AboutFooter

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/frontend/next/src/components/AboutFooter.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { makeStyles, useTheme } from '@material-ui/core/styles';
 import { Box, Divider, Grid, Typography, useMediaQuery } from '@material-ui/core';
-import GitHubIcon from '@material-ui/icons/Github';
+import GitHubIcon from '@material-ui/icons';
```
Comment (tonyvugithub) [inline src/frontend/next/src/components/AboutFooter.tsx:3]: this should be \@material-ui/icons/GitHub'
image

**Review by tonyvugithub — CHANGES_REQUESTED** [review]

File: src/frontend/next/src/components/AboutFooter.tsx (hunk 1 above)
Comment (HyperTHD) [reply src/frontend/next/src/components/AboutFooter.tsx:3]: Fixed, thanks

**Review by tonyvugithub — CHANGES_REQUESTED** [review]
@HyperTHD Can you squash commits into a single one, right now you have 3 commits. Then we are ready to merge, also update the screenshot of the rendered component too.

**Comment by HyperTHD** [comment]
@tonyvugithub I updated the expected output image, the commits can be squashed and merged together so with that, this issue is done

**Comment by tonyvugithub** [comment]
@HyperTHD : How come I still see 3 commits 
image

File: src/frontend/next/public/slack-logo.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (humphd) [inline src/frontend/next/public/slack-logo.svg:1]: We can run this through [svgomg](https://jakearchibald.github.io/svgomg/) and reduce it:

```svg
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="291" height="291"><defs><path d="M.86 184.29c0-16.81 13.73-30.54 30.54-30.54h30.54v30.54c-9.16 20.35-19.34 30.54-30.54 30.54-16.81 0-30.54-13.73-30.54-30.54z" id="a"/><path d="M.86 184.29c0-16.81 13.73-30.54 30.54-30.54h30.54v30.54c-9.16 20.35-19.34 30.54-30.54 30.54-16.81 0-30.54-13.73-30.54-30.54z" id="b"/><path d="M138.39 184.29v76.45c0 16.81-13.73 30.54-30.54 30.54s-30.54-13.73-30.54-30.54v-76.45c0-16.81 13.73-30.54 30.54-30.54s30.54 13.73 30.54 30.54z" id="c"/><path d="M138.39 184.29v76.45c0 16.81-13.73 30.54-30.54 30.54s-30.54-13.73-30.54-30.54v-76.45c0-16.81 13.73-30.54 30.54-30.54s30.54 13.73 30.54 30.54z" id="d"/><path d="M107.85.62c16.81 0 30.54 13.73 30.54 30.53v30.53h-30.54C87.5 52.53 77.31 42.36 77.31 31.15c0-16.8 13.73-30.53 30.54-30.53z" id="e"/><path d="M107.85.62c16.81 0 30.54 13.73 30.54 30.53v30.53h-30.54C87.5 52.53 77.31 42.36 77.31 31.15c0-16.8 13.73-30.53 30.54-30.53z" id="f"/><path d="M107.85 138.14H31.4C14.59 138.14.86 124.41.86 107.6c0-16.8 13.73-30.53 30.54-30.53h76.45c16.81 0 30.54 13.73 30.54 30.53 0 16.81-13.73 30.54-30.54 30.54z" id="g"/><path d="M107.85 138.14H31.4C14.59 138.14.86 124.41.86 107.6c0-16.8 13.73-30.53 30.54-30.53h76.45c16.81 0 30.54 13.73 30.54 30.53 0 16.81-13.73 30.54-30.54 30.54z" id="h"/><path d="M291.52 107.6c0 16.81-13.73 30.54-30.53 30.54h-30.53V107.6c9.15-20.36 19.32-30.53 30.53-30.53 16.8 0 30.53 13.73 30.53 30.53z" id="i"/><path d="M291.52 107.6c0 16.81-13.73 30.54-30.53 30.54h-30.53V107.6c9.15-20.36 19.32-30.53 30.53-30.53 16.8 0 30.53 13.73 30.53 30.53z" id="j"/><path d="M154 107.6V31.15C154 14.35 167.73.62 184.54.62c16.8 0 30.53 13.73 30.53 30.53v76.45c-9.15 20.35-19.33 30.54-30.53 30.54-16.81 0-30.54-13.73-30.54-30.54z" id="k"/><path d="M154 107.6V31.15C154 14.35 167.73.62 184.54.62c16.8 0 30.53 13.73 30.53 30.53v76.45c-9.15 20.35-19.33 30.54-30.53 30.54-16.81 0-30.54-13.73-30.54-30.54z" id="l"/><path d="M184.54 291.28c-16.81 0-30.54-13.73-30.54-30.54V230.2h30.54c20.36 9.16 30.53 19.34 30.53 30.54 0 16.81-13.73 30.54-30.53 30.54z" id="m"/><path d="M184.54 291.28c-16.81 0-30.54-13.73-30.54-30.54V230.2h30.54c20.36 9.16 30.53 19.34 30.53 30.54 0 16.81-13.73 30.54-30.53 30.54z" id="n"/><path d="M184.54 153.75h76.45c16.8 0 30.53 13.73 30.53 30.54s-13.73 30.54-30.53 30.54h-76.45c-20.35-9.16-30.54-19.34-30.54-30.54 0-16.81 13.73-30.54 30.54-30.54z" id="o"/><path d="M184.54 153.75h76.45c16.8 0 30.53 13.73 30.53 30.54s-13.73 30.54-30.53 30.54h-76.45c-20.35-9.16-30.54-19.34-30.54-30.54 0-16.81 13.73-30.54 30.54-30.54z" id="p"/></defs><use xlink:href="#a" fill="#e5e5e5"/><use xlink:href="#a" fill-opacity="0" stroke="#000" stroke-opacity="0"/><use xlink:href="#b" fill="#e5e5e5"/><use xlink:href="#b" fill-opacity="0" stroke="#000" stroke-opacity="0"/><use xlink:href="#c" fill="#e5e5e5"/><use xlink:href="#c" fill-opacity="0" stroke="#000" stroke-opacity="0"/><use xlink:href="#d" fill="#e5e5e5"/><use xlink:href="#d" fill-opacity="0" stroke="#000" stroke-opacity="0"/><use xlink:href="#e" fill="#e5e5e5"/><use xlink:href="#e" fill-opacity="0" stroke="#000" stroke-opacity="0"/><use xlink:href="#f" fill="#e5e5e5"/><use xlink:href="#f" fill-opacity="0" stroke="#000" stroke-opacity="0"/><use xlink:href="#g" fill="#e5e5e5"/><use xlink:href="#g" fill-opacity="0" stroke="#000" stroke-opacity="0"/><use xlink:href="#h" fill="#e5e5e5"/><use xlink:href="#h" fill-opacity="0" stroke="#000" stroke-opacity="0"/><g><use xlink:href="#i" fill="#e5e5e5"/><use xlink:href="#i" fill-opacity="0" stroke="#000" stroke-opacity="0"/></g><g><use xlink:href="#j" fill="#e5e5e5"/><use xlink:href="#j" fill-opacity="0" stroke="#000" stroke-opacity="0"/></g><g><use xlink:href="#k" fill="#e5e5e5"/><use xlink:href="#k" fill-opacity="0" stroke="#000" stroke-opacity="0"/></g><g><use xlink:href="#l" fill="#e5e5e5"/><use xlink:href="#l" fill-opacity="0" stroke="#000" stroke-opacity="0"/></g><g><use xlink:href="#m" fill="#e5e5e5"/><use xlink:href="#m" fill-opacity="0" stroke="#000" stroke-opacity="0"/></g><g><use xlink:href="#n" fill="#e5e5e5"/><use xlink:href="#n" fill-opacity="0" stroke="#000" stroke-opacity="0"/></g><g><use xlink:href="#o" fill="#e5e5e5"/><use xlink:href="#o" fill-opacity="0" stroke="#000" stroke-opacity="0"/></g><g><use xlink:href="#p" fill="#e5e5e5"/><use xlink:href="#p" fill-opacity="0" stroke="#000" stroke-opacity="0"/></g></svg>
```

**Review by humphd — APPROVED** [review]

**Comment by HyperTHD** [comment]
@tonyvugithub This is the option I was talking about. It'll squash and merge the 3 commits together into just 1 
SquashMerge

**Comment by tonyvugithub** [comment]
I see. I actually never used it. Thanks for pointing it out

**Review by tonyvugithub — APPROVED** [review]

**Pull request merged**